jekyll/_cci2/config-policy-management-overview.adoc

```diff
@@ -545,7 +287,7 @@ The CLI provides `policy logs` command to fetch the policy decision logs for you
 
 Following is the output of this command when run with `--help` flag:
 
-# TODO: update help output to contain status filter flag
+#TODO: update help output to contain status filter flag#
```

Comment:
@sagar-connect could you check if this "To do" has been addressed?

Reply:
This has been addressed

```
circleci policy logs --help                                                                                  11:43:35
Get policy decision logs / Get decision log (or policy bundle) by decision ID


Usage:
  circleci policy logs [decision_id] [flags]

Examples:
policy logs --owner-id 462d67f8-b232-4da4-a7de-0c86dd667d3f --after 2022/03/14 --out output.json

Flags:
      --after string        filter decision logs triggered AFTER this datetime
      --before string       filter decision logs triggered BEFORE this datetime
      --branch string       filter decision logs based on branch name
      --context string      policy context (default "config")
  -h, --help                help for logs
      --out string          specify output file name
      --owner-id string     the id of the policy's owner
      --policy-bundle       get only the policy bundle for given decisionID
      --project-id string   filter decision logs based on project-id
      --status string       filter decision logs based on their status

Global Flags:
      --host string              URL to your CircleCI host, also CIRCLECI_CLI_HOST (default "https://circleci.com")
      --policy-base-url string   base url for policy api (default "https://internal.circleci.com")
      --skip-update-check        Skip the check for updates check run before every command.
      --token string             your token for using CircleCI, also CIRCLECI_CLI_TOKEN
```